dev-python/symengine/symengine-9999.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python{2_7,3_3,3_4,3_5} )
+
+inherit git-r3 distutils-r1
+
+DESCRIPTION="Python wrappers to the symengine C++ library"
+HOMEPAGE="https://github.com/sympy/symengine.py"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/symengine/symengine.py.git"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
```

Comment:
This is automatically set by the distutils-r1.eclass. Only python-r1 and python-single need it.